WordPress/gutenberg
# Upgrade react-native-video and drop jetifier on Android

⚠️  Need to update the react-native-dependency to reference a `.tgz` file before merging this.  ⚠️

Related gutenberg-mobile PR: https://github.com/wordpress-mobile/react-native-video/commit/5c6697afc5244883615acc84f054a1ed2a0c6ed0

## Description
This PR updates react-native-video to allow us to drop jetifier on Android.

## Testing Instructions
Using the demo app, play the video from the sample content. On Android this should route to an external application (probably the web browser) and either play the video or offer to download it. On iOS though, the demo video should open and play within the demo app itself.

## Screenshots 

## Types of changes





## Checklist:
